static/app/components/events/groupEventEntries.spec.tsx

```diff
@@ -1,13 +1,15 @@
 import {initializeOrg} from 'sentry-test/initializeOrg';
-import {act, render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+import {act, render, screen, userEvent, within} from 'sentry-test/reactTestingLibrary';
 
 import type {Error} from 'sentry/components/events/errors';
 import EventEntries from 'sentry/components/events/eventEntries';
+import {Group, IssueCategory} from 'sentry/types';
 import {EntryType, Event} from 'sentry/types/event';
 import {OrganizationContext} from 'sentry/views/organizationContext';
 import {RouteContext} from 'sentry/views/routeContext';
 
 const {organization, project, router} = initializeOrg();
+organization.features.push('performance-issues');
```

Comment:
you should be able to pass an org object with features like so:
`inititalizeOrg({organization: {features: ['global-views']}})` so you don't have to push the specific flag